shared/ui/Stream/ConfigureTokenProviderPanel.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from "react";
+import { injectIntl } from "react-intl";
+import { connect } from "react-redux";
+import { closePanel } from "./actions";
+import { configureProvider } from "../store/providers/actions";
+import CancelButton from "./CancelButton";
+import Button from "./Button";
+import { PROVIDER_MAPPINGS } from "./CrossPostIssueControls/types";
+import { Link } from "../Stream/Link";
+
+export class ConfigureTokenProviderPanel extends Component {
+	initialState = {
+		token: "",
+		tokenTouched: false,
+		formTouched: false
+	};
+
+	state = this.initialState;
+
+	focusInput() {
+		document.getElementById("configure-provider-access-token").focus();
```

Comment:
```suggestion

		const el = document.getElementById("configure-provider-access-token");
		el && el.focus();
```

this can throw if the `el` isn't there (can happen if you reload CS with that panel visible)